syntax = "proto3";

option go_package = "gen/go/application";

service ApplicationService {
  rpc CreateXLSXApplications (CreateXLSXApplicationRequest) returns (CreateXLSXApplicationResponse);
}

message CreateXLSXApplicationRequest {
  repeated int32 applications_ids = 1;
}

message CreateXLSXApplicationResponse {
  string xlsx_path = 1;
}
